doc/source/file_formats.rst

```diff
@@ -46,9 +46,48 @@ experimental support for NeuroLucida .asc files.
 SWC
 ---
 
-The SWC format represents a neuron as a tree of 3D points with additional information.
+The SWC format represents a neuron as a set of trees that are connected to a soma.
+The neuronal morphology is encoded as a rooted tree of 3D points and the corresponding radii.
 More information can be found `here <http://research.mssm.edu/cnic/swc.html>`_.
 
+The points of type "1" that represent the soma, have to be in the beginning of the file. 
```

Comment:
The restriction concerns the fact bifurcation of tree from a soma point. I was not aware if the points had to be in the beginning of the file which is something different. After the meeting about reconstructions (with Julian, Rembrandt and Giorgio) I became aware of this restriction.